fortuna/model/model_manager/classification.py

```diff
@@ -69,3 +74,81 @@ def init(
         return dict(
             model=self.model.init(rngs, jnp.zeros((1,) + input_shape), **kwargs)
         )
+
+
+class SNGPModelManager(ClassificationModelManager):
```

Comment:
Perhaps we can call this `SNGPClassificationModelManager`? So far all the implementation is for classification only, but I don't see major blockers for extending the work to regression. In that case, we will need to be able to discriminate b/w regression and classification objects.

If the same consideration applies for other objects you created, please consider adding "Classification" in the name.